usr/local/share/vulture-utils/common.sh

```diff
@@ -58,11 +58,32 @@ exec_mongo() {
     if ! /usr/sbin/jls | /usr/bin/grep -q mongodb; then
         return 1
     fi
-    if [ -z "$_hostname" ] || [ -z "${_command}" ]; then
+    if [ -z "$_hostname" ]; then
         return 1
     fi
 
-    /usr/sbin/jexec mongodb mongo --ssl --sslCAFile /var/db/pki/ca.pem --sslPEMKeyFile /var/db/pki/node.pem "${_hostname}:9091" -eval "${_command}"
+    if [ -z "${_command}" ]; then
+        /usr/sbin/jexec mongodb mongo --quiet --ssl --sslCAFile /var/db/pki/ca.pem --sslPEMKeyFile /var/db/pki/node.pem "${_hostname}:9091/vulture"
+    else
+        /usr/sbin/jexec mongodb mongo --quiet --ssl --sslCAFile /var/db/pki/ca.pem --sslPEMKeyFile /var/db/pki/node.pem "${_hostname}:9091/vulture" --eval "${_command}"
+    fi
```

Comment:
Can you check if this addition suits our needs please ?
I need to point on vulture database for my feature.
I already checked that "db.fsyncLock()" was working as expected even on /vulture.
Do you see any other use case that can conflict with this change ?